<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateECPriKeyDlg</class>
 <widget class="QDialog" name="CreateECPriKeyDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create ec private key</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QComboBox" name="mSlotsCombo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Slot list</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Slot status</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item row="0" column="3">
          <widget class="QLineEdit" name="mSessionText">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>221</red>
                 <green>221</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>221</red>
                 <green>221</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>240</green>
                 <blue>240</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="mSlotIDText">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>221</red>
                 <green>221</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>221</red>
                 <green>221</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>240</green>
                 <blue>240</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QLineEdit" name="mLoginText">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>221</red>
                 <green>221</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>221</red>
                 <green>221</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>240</green>
                 <blue>240</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Session</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Slot ID</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Subject</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="mIDText"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Label</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="mUseSKICheck">
           <property name="text">
            <string>Use SKI</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="mLabelText"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="mSubjectText"/>
         </item>
         <item row="1" column="2">
          <widget class="QComboBox" name="mSubjectTypeCombo"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Private Key</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item row="0" column="3">
          <widget class="QComboBox" name="mParamCombo"/>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="mGenKeyBtn">
           <property name="text">
            <string>Gen Key</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Key Value</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Named Curve</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QPushButton" name="mFindKeyBtn">
           <property name="text">
            <string>Find Key</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item row="1" column="1">
          <widget class="QLineEdit" name="mKeyValueText"/>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="mECParamsText"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>EC Params</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Key value</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="mECParamsLenText">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>221</red>
                 <green>221</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>221</red>
                 <green>221</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>240</green>
                 <blue>240</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="mKeyValueLenText">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>221</red>
                 <green>221</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>221</red>
                 <green>221</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>240</green>
                 <blue>240</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Attributes</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item row="0" column="2">
          <widget class="QCheckBox" name="mDecryptCheck">
           <property name="text">
            <string notr="true">Decrypt</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="mModifiableCheck">
           <property name="text">
            <string notr="true">Modifiable</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QComboBox" name="mDecryptCombo"/>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="mPrivateCombo"/>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="mDeriveCheck">
           <property name="text">
            <string notr="true">Derive</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="mDeriveCombo"/>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="mUnwrapCheck">
           <property name="text">
            <string notr="true">Unwrap</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QCheckBox" name="mSensitiveCheck">
           <property name="text">
            <string notr="true">Sensitive</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="mUnwrapCombo"/>
         </item>
         <item row="1" column="5">
          <widget class="QComboBox" name="mSensitiveCombo"/>
         </item>
         <item row="2" column="5">
          <widget class="QComboBox" name="mTokenCombo"/>
         </item>
         <item row="0" column="5">
          <widget class="QComboBox" name="mSignCombo"/>
         </item>
         <item row="2" column="4">
          <widget class="QCheckBox" name="mTokenCheck">
           <property name="text">
            <string notr="true">Token</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="mPrivateCheck">
           <property name="text">
            <string notr="true">Private</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QCheckBox" name="mSignCheck">
           <property name="text">
            <string notr="true">Sign</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QComboBox" name="mExtractableCombo"/>
         </item>
         <item row="1" column="3">
          <widget class="QComboBox" name="mModifiableCombo"/>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="mExtractableCheck">
           <property name="text">
            <string notr="true">Extractable</string>
           </property>
          </widget>
         </item>
         <item row="3" column="5">
          <widget class="QDateEdit" name="mEndDateEdit"/>
         </item>
         <item row="3" column="4">
          <widget class="QCheckBox" name="mEndDateCheck">
           <property name="text">
            <string>EndDate</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QDateEdit" name="mStartDateEdit"/>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="mStartDateCheck">
           <property name="text">
            <string>StartDate</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="mSignRecoverCheck">
           <property name="text">
            <string>SignRecover</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="mSignRecoverCombo"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>mSlotsCombo</tabstop>
  <tabstop>mSlotIDText</tabstop>
  <tabstop>mSessionText</tabstop>
  <tabstop>mLoginText</tabstop>
  <tabstop>mLabelText</tabstop>
  <tabstop>mSubjectText</tabstop>
  <tabstop>mIDText</tabstop>
  <tabstop>mECParamsText</tabstop>
  <tabstop>mECParamsLenText</tabstop>
  <tabstop>mKeyValueText</tabstop>
  <tabstop>mKeyValueLenText</tabstop>
  <tabstop>mPrivateCheck</tabstop>
  <tabstop>mPrivateCombo</tabstop>
  <tabstop>mDecryptCheck</tabstop>
  <tabstop>mDecryptCombo</tabstop>
  <tabstop>mSignCheck</tabstop>
  <tabstop>mSignCombo</tabstop>
  <tabstop>mUnwrapCheck</tabstop>
  <tabstop>mUnwrapCombo</tabstop>
  <tabstop>mModifiableCheck</tabstop>
  <tabstop>mModifiableCombo</tabstop>
  <tabstop>mSensitiveCheck</tabstop>
  <tabstop>mSensitiveCombo</tabstop>
  <tabstop>mDeriveCheck</tabstop>
  <tabstop>mDeriveCombo</tabstop>
  <tabstop>mExtractableCheck</tabstop>
  <tabstop>mExtractableCombo</tabstop>
  <tabstop>mTokenCheck</tabstop>
  <tabstop>mTokenCombo</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateECPriKeyDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateECPriKeyDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
